src/GitHub.Exports/Models/PullRequestDetailModel.cs

```diff
@@ -93,7 +93,12 @@ public class PullRequestDetailModel
         public IReadOnlyList<PullRequestReviewThreadModel> Threads { get; set; }
 
         /// <summary>
-        /// Gets or sets a collection of pull request Checks & Statuses
+        /// Gets or sets a collection of pull request Checks Suites
+        /// </summary>
+        public List<CheckSuiteModel> CheckSuites { get; set; }
```

Comment:
`IReadOnlyList` is used for the lists elsewhere in this model (and oops I didn't notice this with `Statuses`).